public/javascripts/Admin/src/Admin.LabelSearch.js

```diff
@@ -3,7 +3,7 @@ function AdminLabelSearch() {
 
 
     function _init() {
-        adminGSVLabelView = AdminGSVLabelView(true);
+        adminGSVLabelView = AdminGSVLabelView(true, "AdminLabelSearch");
```

Comment:
Let's standardize the naming convention for the Admin page. I like the names you have outside of /admin. For the admin page, let's go with "AdminMapTab", "AdminContributionsTab", and "AdminLabelSearchTab"!